fossunited/api/pages.py

```diff
@@ -0,0 +1,43 @@
+import frappe
+
+
+@frappe.whitelist(allow_guest=True)
+def search_foss_club(query):
```

Comment:
The function actually searches based ob the given query in the search-component and returns the foss clubs as list based on it. 

Maybe we can make it `search_foss_clubs()` ? as it returns 0 or a list of clubs upon search. Since this change is out of scope in terms of this PR - i think it would be better to change it in a different PR too.

Reply:
`search_foss_clubs` sounds good to me. resolving this conversation.